app/src/main/java/de/westnordost/streetcomplete/MainActivity.java

```diff
@@ -849,6 +869,7 @@ private void showMarkerSolvedAnimation(@DrawableRes int iconResId, PointF startS
 		flingQuestMarkerTo(img, answersCounter.getAnswerTarget(), () -> {
 			root.removeView(img);
 			answersCounter.addOneUnsynced(source);
+			btnUndo.setVisible(true);
```

Comment:
Not all quests can be undone. I.e. note quests can't. Maybe need to call `updateUndoButtonVisibility();` instead, or find out first if the solved quest is something undoable.